### PR TITLE
Better logging with winston

### PR DESCRIPTION
**Pull request by flovilmart**
Centralizes all logs (including verbose) through winston.

Makes all logs available through the API

### CODE REVIEW

File: src/ParseServer.js [hunk 1]
```diff
@@ -254,7 +263,7 @@ class ParseServer {
     if (!process.env.TESTING) {
       process.on('uncaughtException', (err) => {
         if ( err.code === "EADDRINUSE" ) { // user-friendly message for this common error
-          console.log(`Unable to listen on port ${err.port}. The port is already in use.`);
+          log.error(`Unable to listen on port ${err.port}. The port is already in use.`);
```
Comment (drew-gross) [inline src/ParseServer.js]: This one I think actually should be `console.log`

**Comment by drew-gross** [comment]
Looks good to me aside from the one thing. @peterdotjs you wrote most of this code, want to take a look?

File: src/ParseServer.js (hunk 1 above)
Comment (flovilmart) [inline src/ParseServer.js]: alright!

**Comment by peterdotjs** [comment]
LGTM.

**Comment by flovilmart** [comment]
Merge when you like :)

**Pull request merged**